tsconfig.json

```
@@ -1,7 +1,18 @@
 {
-  "extends": "@vercel/style-guide/typescript",
   "compilerOptions": {
-    "target": "ES2019"
+    "moduleResolution": "bundler",
+    "module": "esnext",
+    "target": "ES2019",
```

If we want to target node@18 then we can bump this to es2022

```suggestion
    "target": "es2022",
```